RoddyCore/src/de/dkfz/roddy/config/ToolEntry.java

```diff
@@ -217,6 +219,39 @@ public ToolStringParameter clone() {
         }
     }
 
+    public static class ToolFileParameterCondition {
```

Comment:
Is this really a "condition"? Isn't this rather a "factor" (R-speak) upon which to condition?